From foo@baz Wed Dec 18 20:56:14 PST 2013
From: Simon Horman <horms+renesas@verge.net.au>
Date: Tue,  5 Mar 2013 16:45:15 +0900
Subject: [PATCH 2/3] ARM: shmobile: armadillo800eva: enable NEON on defconfig
To: ltsi-dev@lists.linuxfoundation.org
Cc: Greg KH <gregkh@linuxfoundation.org>
Message-ID: <1362469516-21924-3-git-send-email-horms+renesas@verge.net.au>


From: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>

The application / library will be stopped by illegal instruction
on current Armaddilo800eva board,
since defconfig doesn't have CONFIG_NEON.
This patch enabled it.
Special thanks to Ishiyama-san

Reported-by: Kiyoshi Ishiyama <kiyoshi.ishiyama.wg@renesas.com>
Signed-off-by: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Acked-by: Kiyoshi Ishiyama <kiyoshi.ishiyama.wg@renesas.com>
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
---
 arch/arm/configs/armadillo800eva_defconfig |    1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/configs/armadillo800eva_defconfig
+++ b/arch/arm/configs/armadillo800eva_defconfig
@@ -41,6 +41,7 @@ CONFIG_CMDLINE="console=tty0 console=tty
 CONFIG_CMDLINE_FORCE=y
 CONFIG_KEXEC=y
 CONFIG_VFP=y
+CONFIG_NEON=y
 CONFIG_CMA=y
 # CONFIG_CORE_DUMP_DEFAULT_ELF_HEADERS is not set
 # CONFIG_SUSPEND is not set
